tests/main/views/test_suppliers.py

```diff
@@ -1914,6 +1914,57 @@ def test_set_application_company_details_confirmed_updates_declaration(
         }
 
 
+class TestDeleteUnsuccessfulApplicantDeclarations(BaseApplicationTest, FixtureMixin,
+                                                  PutDeclarationAndDetailsAndServicesMixin):
+    def setup(self):
+        """
+        This sets up the class. It creates a dummy framework and a supplier that applied to that framework, with a
```

Comment:
We don't actually create a dummy framework here - hilariously we just never delete them when we teardown API tests, so they're just 'there'. Obviously not ideal, but let's not make a confusing situation more confusing in this docstring: how about `Set up test data for a supplier and their successful application to a framework.` (We also don't set up the declaration here, but in the test itself)